src/test/operators/index_scan_test.cpp

```diff
@@ -27,15 +29,40 @@ class OperatorsIndexScanTest : public BaseTest {
     TableColumnDefinitions column_definitions;
     column_definitions.emplace_back("a", DataType::Int);
     column_definitions.emplace_back("b", DataType::Int);
-    auto table = std::make_shared<Table>(column_definitions, TableType::Data, 5);
-    for (int i = 0; i <= 24; i += 2) table->append({i, 100 + i});
-    ChunkEncoder::encode_all_chunks(table);
 
-    _chunk_ids = std::vector<ChunkID>(table->chunk_count());
+    // randomized entries
+    auto table_randomized = std::make_shared<Table>(column_definitions, TableType::Data, 5);
+
+    std::random_device rd;
+    std::mt19937 g(rd());
```

Comment:
I am not a big fan of using random in tests. This might have slipped through in other places, but in general it causes issues when two test execution run on different seeds. Theoretically, the PRNG could generate the very same ordered list that hid the issue before. Either explicitly define what order was previously untested or at least choose a fixed seed.